README.md

```diff
@@ -24,23 +24,34 @@ Errors are logged to the DataLab [Sentry](https://sentry.io) account.
 
 ## Local Development Set Up
 ### Native
+
+Prerequisites:
+- **Python v3.9.x**
+- **Pip**
```

Comment:
Nit: probably virtualenv too?  I'm not sure all pythons come with `venv` installed.